1_fetch.R

```diff
@@ -0,0 +1,33 @@
+source("1_fetch/src/get_nwis_data.R")
+
+p1_targets_list <- list(
+  tar_target(
+    nwis_01427207_data,
+    download_nwis_site_data_object("01427207")
+  ),
```

Comment:
The USGS convention when using the phases approach is to also prefix the target names with their associate phase. So here in phase 1 you would have `p1_` in front of the target names.

Reply:
Great thanks for catching that. I just updated it with the last commit. I'll go ahead and merge if that works for you. Thanks!